pages/landing/enterprise-eks/index.html

```diff
@@ -0,0 +1,191 @@
+---
+layout: landing
+type: landing
+slug: index-page
+
+title: Enterprise apps on Amazon EKS. In about a day.
+excerpt: Build battle-tested Amazon EKS clusters, designed for enterprise with security, observability, and usability built-in. Manage 100% of your infrastructure using code, all in ~1 day. <b>Join now for special early access pricing!</b>
+permalink: /eks-for-enterprise/
+custom_js:
+- prism
+prism_css: /assets/css/prism-landing-zone.css
+
+technologies:
+- name: Spring Boot
+  logo: /assets/img/logos-technologies/logo-spring-boot.png
+- name: Ruby on Rails
+  logo: /assets/img/logos-technologies/logo-rails.png
+
+hero:
+  image: /assets/img/infrastructure-cube-icons/infrastructure-in-a-day.png
+  cta: Request early access
+  logos:
+  - /assets/img/logos-technologies/partner_logo_aws.png
+  - /assets/img/logos-technologies/partner_logo_kubernetes.png
+  - /assets/img/logos-technologies/partner_logo_terraform.png
+
+how_it_works:
+  - heading: "Step 1: Choose your EKS configuration"
+    precode: |
+      <figure><figcaption>eks-configuration.yml</figcaption>
+        <pre>
+            <code class="language-yml">
+            # Configure CIS Benchmark compliance
+            cis:
+              enabled: true
+
+            # Choose from istio, consul-connect, aws-app-mesh
+            service_mesh: "istio"
+
+            # Choose from flux, jenkins, argocd
+            cicd: "flux"
+
+            # Choose from prometheus, datadog, cloudwatch
+            observability: prometheus
```

Comment:
Observability seems a bit of a broad term here. What are we picking? Where to send metrics? Logs? What to use for alerts?

Reply:
This has been replaced with `metric_aggregation` and `log_aggregation`.